lib/jekyll/commands/new_theme.rb

```diff
@@ -22,14 +22,20 @@ def process(args, opts)
       end
 
       new_theme_name = args.join("_")
-      theme = Jekyll::ThemeBuilder.new(new_theme_name, opts)
+      @theme = Jekyll::ThemeBuilder.new(new_theme_name, opts)
+
+      update_logger
+    end
+
+    def update_logger
```

Comment:
I agree, it was added to satisfy rubocop with an earlier commit. I forgot to remove it when I reverted back to using interpolation within the strings.